DevoxxClientMobile/src/main/java/com/devoxx/views/FeedbackPresenter.java

```diff
@@ -0,0 +1,129 @@
+import javafx.css.PseudoClass;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+
+import javax.inject.Inject;
+
+public class FeedbackPresenter extends GluonPresenter<DevoxxApplication> {
+
+    private static final PseudoClass PSEUDO_CLASS_ERROR = PseudoClass.getPseudoClass("error");
+
+    @FXML
+    private View feedback;
+
+    @FXML
+    private TextField name;
+    
+    @FXML
+    private TextField email;
+    
+    @FXML
+    private TextArea message;
+    
+    @Inject
+    private Service service;
+    
+    public void initialize() {
+        
+        feedback.setOnShowing( event -> {
+            final User authenticatedUser = service.getAuthenticatedUser();
+            if (authenticatedUser != null) {
+                name.setText(authenticatedUser.getName());
+                email.setText(authenticatedUser.getEmail());
+            }
+            
+            final Button close = MaterialDesignIcon.CLEAR.button(e -> DevoxxView.SESSIONS.switchView());
+            final Button send = MaterialDesignIcon.SEND.button(e -> {
+                final String nameText = name.getText();
+                final String emailText = email.getText();
+                final String messageText = message.getText();
+                if (validateInput(name, email, message)) {
+                    sendFeedback(nameText, emailText, messageText);
+                }
+            });
+            AppBar appBar = getApp().getAppBar();
+            appBar.setNavIcon(close);
+            appBar.setTitleText(DevoxxView.FEEDBACK.getTitle());
+            appBar.getActionItems().add(send);
+        });
+        
+        feedback.setOnShown(e -> feedback.requestFocus());
+        addValidator(name, email, message);
+    }
+
+    private void sendFeedback(String nameText, String emailText, String messageText) {
+        final Feedback feedback = new Feedback(nameText, emailText, messageText);
+        service.submitFeedback(feedback);
+        final Toast toast = new Toast(DevoxxBundle.getString("OTN.FEEDBACK.SUCCESS"));
+        toast.show();
+        DevoxxView.SESSIONS.switchView();
+    }
+
+    private boolean validateInput(TextInput... textInputs) {
+        boolean status = true;
+        for (TextInput textInput : textInputs) {
+            if (textInput.getText().isEmpty()) {
+                textInput.pseudoClassStateChanged(PSEUDO_CLASS_ERROR, true);
+                status = status && false;
+            } else {
+                textInput.pseudoClassStateChanged(PSEUDO_CLASS_ERROR, false);
+                status = status && true;
+            }    
+        }
+        return status;
+    }
+
+    private void addValidator(TextInput... textInputs) {
+        for (TextInput textInput : textInputs) {
+            if (textInput.getText().isEmpty()) {
+                textInput.setErrorValidator(s -> {
+                    if (s.isEmpty()) {
+                        return "Cannot be empty";
```

Comment:
The validation on the textArea is not styled.

Reply:
The border was turning red. I should have added it to the float text as well.

Reply:
The issue affects also to the error text